build.sbt

```diff
@@ -1109,10 +1092,6 @@ lazy val mkQuick = taskKey[File]("Generate a full build, including scripts, in b
 lazy val mkPack = taskKey[File]("Generate a full build, including scripts, in build/pack")
 lazy val testAll = taskKey[Unit]("Run all test tasks sequentially")
 
-lazy val testRun = taskKey[Unit]("Run compute intensive test tasks sequentially")
-lazy val testPosPres = taskKey[Unit]("Run compilation test (pos + presentation) sequentially")
-lazy val testRest = taskKey[Unit]("Run the remaining test tasks sequentially")
```

Comment:
I wasn't aware these existed. Do we know who added them?

Reply:
Adriaan did, while setting up Travis: #6390. I think the idea was to run the tests one after another somehow?

Reply:
#6630 removed them from travis.yml; for some reason they're not in 2.13.x